src/Adapter/Module/Tab/ModuleTabRegister.php

```diff
@@ -80,22 +81,36 @@ class ModuleTabRegister
      */
     private $languages;
 
+    /**
+     * @var LoaderInterface
+     */
+    private $routingConfigLoader;
+
     /**
      * @param TabRepository $tabRepository
      * @param LangRepository $langRepository
      * @param LoggerInterface $logger
      * @param TranslatorInterface $translator
      * @param Filesystem $filesystem
      * @param array $languages
+     * @param LoaderInterface $routingConfigLoader
      */
-    public function __construct(TabRepository $tabRepository, LangRepository $langRepository, LoggerInterface $logger, TranslatorInterface $translator, Filesystem $filesystem, array $languages)
```

Comment:
BC break.
But it's a class highly related to internal behavior so I think it's all right.

However we should acknowledge it:
- mark "yes" to "BC breaks ?" in PR body
- add "BC break" label